web/packages/teleport/src/UnifiedResources/UnifiedResources.tsx

```diff
@@ -101,6 +102,12 @@ function ClusterResources({
   const teleCtx = useTeleport();
   const flags = teleCtx.getFeatureFlags();
 
+  const { setEnforceMinWidth } = useContext(ContentMinWidthContext);
+
+  useEffect(() => {
+    setEnforceMinWidth(false);
```

Comment:
This works only one-way.
If you go to the unified view and then to any other page, the `enforceMinWidth` will be still `false`.

A potential fix for this would be to add a cleanup function to the `useEffect` that would set `setEnforceMinWidth(true)` . 

------
But TBH, it seems to me that creating just one responsive page in the app isn't that useful for the user. If you navigate to any other page, you will need to expand the window. Otherwise you will have to scroll horizontally.

I think we should aim for removing the global min-width. Since the test plan starts soon, we could test the entire app for any regressions and then backport the PR to v14.

Reply:
> But TBH, it seems to me that creating just one responsive page in the app isn't that useful for the user.

It would be ideal to have the whole app be responsive, but the resource page is the place more users are going to be most often and where the lack of responsiveness is probably the most cumbersome. I think there's a big UX improvement in making this responsive, even if other parts of the app aren't yet.

> I think we should aim for removing the global min-width. Since the test plan starts soon, we could test the entire app for any regressions and then backport the PR to v14.

I don't think such a global change is worth it right now imo since in v15 we're removing the side nav entirely (which was part of the reason why we even needed the `min-width`) and replacing it with a top bar. I think that would be a much better time to revisit the global `min-width` (and probably be able to just remove it without introducing any bugs). We won't actually be able to meaningfully test for any v14 regressions in the v15 test plan since the layouts will be different. 

cc @avatus

Reply:
>I don't think such a global change is worth it right now imo since in v15 we're removing the side nav entirely

Ok, it sounds like a good opportunity to handle this issue too.